src/Microsoft.Health.Fhir.Api/Features/Context/FhirRequestContextMiddleware.cs

```diff
@@ -13,6 +13,8 @@ namespace Microsoft.Health.Fhir.Api.Features.Context
 {
     public class FhirRequestContextMiddleware
     {
+        private const string RequestIdHeaderName = "X-Request-Id";
```

Comment:
We have some headers that are in all lower cases (x-ms-session-token, x-ms-consistency-level) and some that are in Pascal casing (X-Request-Id, X-Content-Type-Options). Since the X-Request-Id is already being exposed in the PaaS side, I used that but should we standardize them?